.github/workflows/main.yml

```diff
@@ -66,21 +64,9 @@ jobs:
       fail-fast: false
       matrix:
         ceph_version:
-        - "octopus"
-        - "pacific"
-        - "quincy"
-        - "reef"
-        - "squid"
-        - "pre-reef"
-        - "pre-squid"
-        - "main"
+        - wip-pkalever-rbd-group-snap-mirror
```

Comment:
Will this stay available, or does it get garbage collected?

Reply:
No, it will be garbage collected.
Maybe we can cache it.
I came across this [action](https://github.com/marketplace/actions/docker-cache) for caching images, let me know if there is something else better suited?
Though, I had one question, that if the image changes, then will it still be a `cache-hit` or a `cache-miss`, considering the image tag hasn't updated 🤔

Reply:
Just copy the image to a personal quay repository maybe? That is probably the easiest for now.

Reply:
Even that would be fetched every time, right?
Or did you mean in conjunction with cache action?

Reply:
Yes, it would be fetched every time. Maybe just push it to ghcr.io (and not quay) so it is somewhat local to GitHub? CI would not run very often on this branch, I don't know if caching would be really beneficial.

Reply:
Oh okay.
Pushed the image under my registry and made it public.